pegasus/emails/dashboard.html

```diff
@@ -1,5 +1,11 @@
 ---
-from: '"Code.org" <noreply@code.org>'
+<%
+formatted_from = "'<noreply@code.org>'"
+if defined? from && from != nil
+  formatted_from = "'<" + from + ">'"
+end
+%>
+from: <%= formatted_from %>
```

Comment:
> Should we keep the "Code.org" name? Or have the caller format the from address so they can add a name if the want?

For now I'm going to just add the "Code.org" name to both. Later we can maybe add the ability to specify a name.